### ISSUE TITLE
I can't upload levels.

### ISSUE THREAD

**Issue by Touchcreator** [Cvolton/GMDprivateServer#1041]
I made a 1.2 GDPS and I am unable to upload any levels to it. It says "Upload failed. Please try again later."

**Comment by Cvolton** [comment]
did u enable unregistered submissions

**Comment by Touchcreator** [comment]
> did u enable unregistered submissions

yeah i fixed it lol, but i noticed that my data in the gdps doesn't save, so when i close the app everything is gone
